app.js

```diff
@@ -1,17 +1,31 @@
-const menu = document.querySelector(".menu");
-const hamburger = document.querySelector(".hamburger");
-const closeIcon = document.querySelector(".closeIcon");
-const menuIcon = document.querySelector(".menuIcon");
-closeIcon.style.display = "none";
+const menu = document.querySelector('.menu');
+const hamburger = document.querySelector('.hamburger');
+const closeIcon = document.querySelector('.closeIcon');
+const menuIcon = document.querySelector('.menuIcon');
+closeIcon.style.display = 'none';
 function toggleMenu() {
-  if (menu.classList.contains("showMenu")) {
-    menu.classList.remove("showMenu");
-    closeIcon.style.display = "none";
-    menuIcon.style.display = "block";
+  if (menu.classList.contains('showMenu')) {
+    menu.classList.remove('showMenu');
+    closeIcon.style.display = 'none';
+    menuIcon.style.display = 'block';
   } else {
-    menu.classList.add("showMenu");
-    closeIcon.style.display = "flex";
-    menuIcon.style.display = "none";
+    menu.classList.add('showMenu');
+    closeIcon.style.display = 'flex';
+    menuIcon.style.display = 'none';
   }
 }
-hamburger.addEventListener("click", toggleMenu);
+hamburger.addEventListener('click', toggleMenu);
+
+// Popup Window
+
+const open = document.getElementById('open');
+const modal = document.getElementById('modal-container');
+const close = document.getElementById('close');
+
+open.addEventListener('click', () => {
+  modal.classList.add('show');
+});
```

Comment:
- [ ] The JavaScript should contain an array of objects that will be used to render data in the popup window.
_Here's what it should look like:_
```javascript
const projects = [{
    id: 1
    name: "Example",
    description: "Adescription.",
    image: "link_to_image.jpg",
    technologies: ["html", "JavaScript", "css"],
    link: "link_to_live_version",
}]
```
- You can use the array filter method to filter the object you'll need to use to render data.

Good luck 😄

Reply:
Thank you for your feedback. I have completed that requirement